### ISSUE TITLE
Assessment Results

### ISSUE THREAD

**Issue by francepack** [raz-joan/mod_0_assessment#1]
@raz-joan 
Nice work here- I can tell you've been working with the material taught through Mod0. You are officially `technical ready`. Congrats! A little feedback:

- Very nice use of naming patterns with your boolean and array key names.
- Good job making all of your key ideas relate to the thing you are describing.
- Solid use of a key that had a value of an object! Great practice, and I like how you paid attention to your spacing/indentation. 

Well done! Let me know if you have any questions.

**Comment by raz-joan** [comment]
Awesome news and thanks for the feedback Mason! Just want to clarify, this means I am cleared to start Mod 1 with the 2108 cohort?

Best--  Joan

**Comment by francepack** [comment]
Yes- my records show we have you slated to begin with the next cohort, provided completion of prework (though, usually there is opportunity to improve that if changes are requested).
